### PR TITLE
[service.tvh.manager] 0.3.20

### PR DESCRIPTION
**Pull request by b-jesch**
Description
-----------

Addon for 'PVR Timer- and Powermanagement' for XBMC/Kodi. This addon turns your vanilla Kodi on a Linux installation (pure Linux, Kodibuntu, Open-/LibreELEC) into a full featured video recorder.

This addon handles poweroff management for current active recordings and wakeup procedures for future schedules using the XML-Interface of TVHeadend. This plugin starts and shut down the htpc if a recording is scheduled. It delivers optional Emails via SMTP if an automatic scheduled Recording ended successfully.

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [sciprt.foo.bar] v1.0.0

Additional information :

service.tvh.manager 0.3.20 initial commit

### CODE REVIEW

File: service.tvh.manager/addon.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<addon id="service.tvh.manager" name="TVHeadend Power and Recording Management" version="0.3.20" provider-name="Birger Jesch">
+  <requires>
+    <import addon="xbmc.python" version="2.20.0" />
+    <import addon="xbmc.addon" version="15.0.0" />
+  </requires>
+  <extension point="xbmc.service" library="default.py" start="[login|startup]" />
+  <extension point="xbmc.python.script" library="default.py" />
+  <extension point="xbmc.addon.metadata">
+    <platform>linux</platform>
+    <language>en de pl</language>
```
Comment (enen92) [inline service.tvh.manager/addon.xml]: Please leave this one blank, it is only applied to the content

File: service.tvh.manager/addon.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <disclaimer lang="de">Benutzung auf eigene Gefahr. Keine Haftung für nicht gesetzte Timer, nicht durchgeführte Aufnahmen und andere Seiteneffekte!</disclaimer>
+    <disclaimer lang="en">Use at your own risk. No liability for not set timers, lack of recording and other side effects!</disclaimer>
+    <disclaimer lang="pl">Używasz na własne ryzyko. Nie ponosimy odpowiedzialności za nieprawidłowe nastawienia czasu nagrania, braku nagrania i innych skutków ubocznych!</disclaimer>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
+    <forum>http://forum.xbmc.org/showthread.php?tid=115758</forum>
+    <website>http://vdr4bj1.no-ip.org</website>
+    <email>birger.jesch@gmail.com</email>
+    <source>https://github.com/b-jesch/service.tvh.manager</source>
+    <assets>
+      <icon>resources/icon.png</icon>
```
Comment (enen92) [inline service.tvh.manager/addon.xml]: You can also ditch the icon.png and the fanart.jpg that are in this folder. Just use the ones in the root folder and remove the resources path from the image definition here. That way you don't have twice the same images.

File: service.tvh.manager/addon.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.tvh.manager" name="TVHeadend Power and Recording Management" version="0.3.20" provider-name="Birger Jesch">
+  <requires>
+    <import addon="xbmc.python" version="2.20.0" />
+    <import addon="xbmc.addon" version="15.0.0" />
```
Comment (MartijnKaijser) [inline service.tvh.manager/addon.xml]: Drop this dependency

**Comment by MartijnKaijser** [comment]
Are the used images licenced in any way like creative Commons? Please specify this somewhere and make sure the license it's compatible with gpl2 

Our default language is English so imo the screenshots should be that as well

File: service.tvh.manager/addon.xml (hunk 3 above)
Comment (b-jesch) [inline service.tvh.manager/addon.xml]: Both (python and addon)?

**Comment by b-jesch** [comment]
Images (icon, fanart) are changed and created by myself.

**Comment by b-jesch** [comment]
All changes done

**Comment by b-jesch** [comment]
Last commit was done automatically by PyCharm... Sorry

**Pull request merged**